packages/components/stories/mobile-drawer/README.md

```diff
@@ -0,0 +1,281 @@
+# MobileDrawer Component
+
+Renders humberger menu in mobile view.
+
+#### Supported Gestures:
+
+-   Click
+
+## Usage
+
+```jsx
+import { MobileDrawer } from 'deriv-components';
+
+const DummyComponent = props => (
+    <MobileDrawer
+        alignment='left'
+        is_open
+        title='Menu'
+        height='100vh'
+        width='295px'
+    >       
+        <MobileDrawer.Body>
+            <MobileDrawer.Item>
+                <p>Menu item 1</p>
+            </MobileDrawer.Item>
+            <MobileDrawer.Item>
+                <p>Menu item 2</p>
+            </MobileDrawer.Item>
+        </MobileDrawer.Body>
+    </MobileDrawer>
+);
+```
+
+
+## Props
+
+| Name                  | Type                | Default   | Description                                         |
+| --------------------- | ------------------- | ----------| --------------------------------------------------- |
+| children              | {react node}        | null      | Menu items, Header, Footer and etc.                 |
+| alignment             | {string}            | `'left'`  | Menu alignment (`'left'` or `'right'`)              |
+| id                    | {string}            | null      | `id` of the main menu                               |
+| is_open `*required`   | {boolean}           | null      | State of the menu on first load                     |
+| className `*optional` | {string}            | null      | ClassName for menu                                  |
+| toggle `*required`    | {function}          | undefined | Callback function for setting `is_open` property    |
+| title                 | {string or boolean} | null      | Title of the menu or `false` for no title           |
+| height                | {string}            | `'auto'`  | Height of the main menu                             |
+| width                 | {string}            | `'auto'`  | Width of the main menu                              |
+| livechat              | {react node}        | null      | LiveChat element will show in header of menu        |
```

Comment:
```suggestion
```

Live chat is not part of the component.

Reply:
I saw that `livechat` is not in the `propTypes` of drawer but appears in props parameter of the main `MobileDrawer` function so I added it in the table. Also, livechat component passed to MobileDrawer here: https://github.com/binary-com/deriv-app/blob/af6e55d41f0c846014ef44cdf9786b2f01ee0073/packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx#L255
Could you please give me more detail?

Reply:
I'm not sure why `livechat` is a prop? 😅 IMO, the `livechat` element should be passed from outside the component since this should only be UI component.
@ali-fs , we'll probably refactor this to make the use-case of this prop more general and not only specific to livechat. So let's not add `livechat` to the storybook documentation in any case.

Reply:
Fixed.